tests.js

```diff
@@ -3,36 +3,19 @@ expect = require("chai").expect;
 should = require("chai").should;
 require('dotenv').load();
 
-var Sequelize = require('sequelize');
-var sequelize = new Sequelize(
-    process.env.PG_DB_TEST,
-    process.env.PG_USER,
-    process.env.PG_PASSWORD, {
-    dialect: "postgres",
-    logging: false
-});
-
-var models = require('./models/db')(sequelize);
-
-var seq_test = function (next) {
-  return function () {
-    beforeEach(function (done) {
-        sequelize.sync({ force: true }).then(function() {
-            done();
-        });
-    });
+// Sequelize mocking
+seq_test = require("./testutils/seq_utils").seq_test;
+models = require("./testutils/seq_utils").models;
 
-    afterEach(function (done) {
-        sequelize.drop().then(function() {
-            done();
-        });
-    });
+// Jsdom mocking
+jsdom_test = require("./testutils/jsdom_utils");
 
-    next();
-  };
-}
-
-describe("Model Unittests", seq_test(function () {
-  require("./models/tests/test_user.js")(models);
-  require("./models/tests/test_interest.js")(models);
+// Tests
+describe("Sequelize Model Unittests", seq_test(function () {
+  require("./models/tests/test_user")(models);
+  require("./models/tests/test_interest")(models);
 }));
+
+describe("React Component Unittests", function () {
+  jsdom_test(require("./src/jsx/ChatBox/test_message.jsx"));
```

Comment:
yes, but i don't think we want to. we probably don't want every test to be treated the same way. some might rely on other mocks. this way we can also see every test we run for easy exclusion and ordering.